packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -194,7 +194,7 @@ export class FeatureDevController {
     // TODO add type
     private async processUserChatMessage(message: any) {
         if (message.message === undefined) {
-            this.messenger.sendErrorMessage('chatMessage should be set', message.tabID, 0, undefined)
+            this.messenger.sendErrorMessage('chatMessage should be set', message.tabID, 0, undefined, undefined)
```

Comment:
non-blocker: I wonder if session can be obtained before this line and the conversationId would be incliuded in this error, too. 

Something we can discuss offline, not a big deal atm.